src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java

```diff
@@ -160,7 +162,8 @@ private Map<String, SonarProjectDto> downloadProjectList(ServerConnection select
     try {
       return ProgressManager.getInstance().run(downloadTask);
     } catch (Exception e) {
-      var msg = e.getMessage() != null ? e.getMessage() : "Failed to download list of projects";
+      var msg = "Failed to download list of projects. Please check the logs for more details.";
+      getService(project, SonarLintConsole.class).debug(e.getMessage());
```

Comment:
I would log this as error, otherwise most users won't see the logs (if it's debug, you have to enable specific options to have them)